server/src/test/java/io/unitycatalog/server/base/schema/BaseSchemaCRUDTest.java

```diff
@@ -84,6 +83,16 @@ public void testSchemaCRUDL() throws ApiException {
     assertThat(updatedSchemaInfo2.getFullName()).isEqualTo(TestUtils.SCHEMA_NEW_FULL_NAME);
     assertThat(updatedSchemaInfo2.getUpdatedAt()).isNotNull();
 
+    // Update schema properties without updating name and comment
+    System.out.println("Testing update schema: changing properties..");
+    UpdateSchema updateSchema3 = new UpdateSchema().properties(TestUtils.NEW_PROPERTIES);
+    schemaOperations.updateSchema(TestUtils.SCHEMA_NEW_FULL_NAME, updateSchema3);
+    SchemaInfo retrievedSchemaInfo3 = schemaOperations.getSchema(TestUtils.SCHEMA_NEW_FULL_NAME);
```

Comment:
Rather than changing the naming convention - please rename `retrievedSchemaInfo` to `updatedSchemaInfo3` and change the name of the occurrence at line 99 to `updatedSchemaInfo4` for consistency. 

Also, for other changes related to this file including the TODO - you may refer [this](https://github.com/unitycatalog/unitycatalog/pull/338/files#diff-aaa8d031b8b88f3e944c9409431246ca3b2ddc9a9b438415b067faa649ed3505).